staging/operator-registry/pkg/registry/types.go

```diff
@@ -289,6 +301,25 @@ func (pd *PackageDependency) Validate() []error {
 	return errs
 }
 
+// Validate will validate constraint type and return error(s)
+func (cc *CelConstraint) Validate() []error {
+	errs := []error{}
+	if cc.Cel == nil {
```

Comment:
@dinhxuanvu 
If using bellow format, error "The CEL field is missing" will be raised.

dependencies:
- type: olm.constraint
  value:
    any:
      constraints:
      - message: Package prometheus is needed for...
        package:
          packageName: prometheus
          versionRange: '>=0.27.0'
      - gvk:
          group: camel.apache.org
          kind: Integration
          version: v1
        message: GVK Integration is needed for
    message: All are required for etcd because..